apps/events/tasks.py

```diff
@@ -12,8 +12,11 @@
 @shared_task(ignore_result=True)
 def enqueue_static_triggers(session_id, trigger_type):
     session = ExperimentSession.objects.get(id=session_id)
+    experiment_id = session.experiment_id
+    if session.experiment.version_number:
+        experiment_id = session.experiment.working_experiment
```

Comment:
If we're copying the triggers shouldn't this load the ones related to the versioned experiment?